materialize-bigquery/.snapshots/TestValidateAndApply

```diff
@@ -87,7 +87,7 @@ Big Schema Changed Types Constraints:
 {"Field":"multipleField","Type":3,"TypeString":"LOCATION_RECOMMENDED","Reason":"This location is part of the current materialization"}
 {"Field":"nullField","Type":4,"TypeString":"FIELD_OPTIONAL","Reason":"Object fields may be materialized"}
 {"Field":"numField","Type":6,"TypeString":"UNSATISFIABLE","Reason":"Field 'numField' is already being materialized as endpoint type 'FLOAT' but endpoint type 'BOOLEAN' is required by its schema '{ type: [boolean] }'"}
-{"Field":"objField","Type":3,"TypeString":"LOCATION_RECOMMENDED","Reason":"This location is part of the current materialization"}
+{"Field":"objField","Type":6,"TypeString":"UNSATISFIABLE","Reason":"Field 'objField' is already being materialized as endpoint type 'JSON' but endpoint type 'STRING' is required by its schema '{ type: [string] }'"}
```

Comment:
is this expected?

Reply:
Yeah, these snapshots are kind of confusing and I should probably change them. But what this is testing is that a field `objField` that was originally a `type: object` can't have it's type changed to `type: string` now. It actually used to be able to since `type: object` was materialized as a string, so changing it to `type: string` was fine.